bika/lims/browser/js/coffee/bika.lims.bikalisting.coffee

```diff
@@ -380,6 +380,7 @@ class window.BikaListingTableView
     form_data.set "#{form_id}_review_state", state_id
 
     @ajax_submit
+      url: window.location.href.split("?")[0]
```

Comment:
There is also a `get_base_url` method, that does exactly this;) Try to read first the code that exists, especially when it has been already refactored.

Reply:
@ramonski I see it now... Would you like me to change it? Sorry I missed that, I can update it without problems.

Reply:
Done in #718